boom! testing td_rredis-0.1.2 against stable for nightly
boom! running: cargo +stable build --frozen
boom! creating container for: cargo +stable build --frozen
boom! running `docker create -v /home/ec2-user/cargobomb/./work/local/test-source/nightly/stable:/source:ro -v /home/ec2-user/cargobomb/./work/local/cargo-home:/cargo-home:ro -v /home/ec2-user/cargobomb/./work/local/rustup-home:/rustup-home:ro -v /home/ec2-user/cargobomb/./work/local/target-dirs/nightly/stable:/target:rw -e USER_ID=500 -e CMD=cargo +stable build --frozen cargobomb`
blam! 2c05791bbdd9f2136d256ca41ebf7e751beacc14d2f0c1320a07f6ee892e50dd
boom! running `docker start -a 2c05791bbdd9f2136d256ca41ebf7e751beacc14d2f0c1320a07f6ee892e50dd`
kablam!    Compiling td_rredis v0.1.2 (file:///source)
kablam! warning: unreachable expression, #[warn(unreachable_code)] on by default
kablam!    --> src/parser.rs:105:26
kablam!     |
kablam! 105 |                 Ok(_) => return fail!((ErrorKind::PatternError, "Could not read enough bytes")),
kablam!     |                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
kablam! 
kablam! warning: unreachable expression, #[warn(unreachable_code)] on by default
kablam!    --> src/lib.rs:7:23
kablam!     |
kablam! 7   |           if !($expr) { return $err_result; }
kablam!     |                         ^
kablam!     | 
kablam!    ::: src/script.rs
kablam!     |
kablam! 173 |                           ensure!(self.script.hash == hash,
kablam!     |  _________________________- starting here...
kablam! 174 | |                                 fail!((ErrorKind::BusyLoadingError, "load hash error")));
kablam!     | |_________________________________________________________________________________________- ...ending here: in this macro invocation
kablam! 
kablam! warning: unreachable expression, #[warn(unreachable_code)] on by default
kablam!    --> src/lib.rs:7:23
kablam!     |
kablam! 7   |           if !($expr) { return $err_result; }
kablam!     |                         ^
kablam!     | 
kablam!    ::: src/script.rs
kablam!     |
kablam! 184 |                           ensure!(self.script.hash == hash,
kablam!     |  _________________________- starting here...
kablam! 185 | |                                 fail!((ErrorKind::BusyLoadingError, "load hash error")));
kablam!     | |_________________________________________________________________________________________- ...ending here: in this macro invocation
kablam! 
kablam!     Finished debug [unoptimized + debuginfo] target(s) in 6.29 secs
boom! running `docker rm -f 2c05791bbdd9f2136d256ca41ebf7e751beacc14d2f0c1320a07f6ee892e50dd`
blam! 2c05791bbdd9f2136d256ca41ebf7e751beacc14d2f0c1320a07f6ee892e50dd
boom! running: cargo +stable test --frozen --no-run
boom! creating container for: cargo +stable test --frozen --no-run
boom! running `docker create -v /home/ec2-user/cargobomb/./work/local/test-source/nightly/stable:/source:ro -v /home/ec2-user/cargobomb/./work/local/cargo-home:/cargo-home:ro -v /home/ec2-user/cargobomb/./work/local/rustup-home:/rustup-home:ro -v /home/ec2-user/cargobomb/./work/local/target-dirs/nightly/stable:/target:rw -e USER_ID=500 -e CMD=cargo +stable test --frozen --no-run cargobomb`
blam! 91b4da1b3c0e901dec4b3b85ec79331eb6113ed40d1d7313d68603e3e5db3454
boom! running `docker start -a 91b4da1b3c0e901dec4b3b85ec79331eb6113ed40d1d7313d68603e3e5db3454`
kablam!    Compiling td_rredis v0.1.2 (file:///source)
kablam! warning: unused import: `std::error`, #[warn(unused_imports)] on by default
kablam!  --> examples/redis.rs:2:5
kablam!   |
kablam! 2 | use std::error;
kablam!   |     ^^^^^^^^^^
kablam! 
kablam! warning: unused import: `std::fmt`, #[warn(unused_imports)] on by default
kablam!  --> examples/redis.rs:3:5
kablam!   |
kablam! 3 | use std::fmt;
kablam!   |     ^^^^^^^^
kablam! 
kablam! warning: unused import: `std::io`, #[warn(unused_imports)] on by default
kablam!  --> examples/redis.rs:4:5
kablam!   |
kablam! 4 | use std::io;
kablam!   |     ^^^^^^^
kablam! 
kablam! warning: unused import: `std::hash::Hash`, #[warn(unused_imports)] on by default
kablam!  --> examples/redis.rs:5:5
kablam!   |
kablam! 5 | use std::hash::Hash;
kablam!   |     ^^^^^^^^^^^^^^^
kablam! 
kablam! warning: unused imports: `Utf8Error`, `from_utf8`, #[warn(unused_imports)] on by default
kablam!  --> examples/redis.rs:6:16
kablam!   |
kablam! 6 | use std::str::{from_utf8, Utf8Error};
kablam!   |                ^^^^^^^^^  ^^^^^^^^^
kablam! 
kablam! warning: unused imports: `HashMap`, `HashSet`, #[warn(unused_imports)] on by default
kablam!  --> examples/redis.rs:7:24
kablam!   |
kablam! 7 | use std::collections::{HashMap, HashSet};
kablam!   |                        ^^^^^^^  ^^^^^^^
kablam! 
kablam! warning: unused import: `std::convert::From`, #[warn(unused_imports)] on by default
kablam!  --> examples/redis.rs:8:5
kablam!   |
kablam! 8 | use std::convert::From;
kablam!   |     ^^^^^^^^^^^^^^^^^^
kablam! 
kablam! warning: unused import: `redis::RedisResult`, #[warn(unused_imports)] on by default
kablam!   --> examples/redis.rs:10:5
kablam!    |
kablam! 10 | use redis::RedisResult;
kablam!    |     ^^^^^^^^^^^^^^^^^^
kablam! 
kablam! warning: unused imports: `FromRedisValue`, `RedisResult`, #[warn(unused_imports)] on by default
kablam!  --> examples/cluster.rs:4:13
kablam!   |
kablam! 4 | use redis::{RedisResult, FromRedisValue};
kablam!   |             ^^^^^^^^^^^  ^^^^^^^^^^^^^^
kablam! 
kablam! warning: unused variable: `xx1`, #[warn(unused_variables)] on by default
kablam!   --> examples/cluster.rs:30:14
kablam!    |
kablam! 30 |     let (xx, xx1) : (String, String) = try!(redis::cmd("MGET").arg(&["key1", "key5"]).query_cluster(&mut cluster));
kablam!    |              ^^^
kablam! 
kablam! warning: unreachable expression, #[warn(unreachable_code)] on by default
kablam!    --> src/parser.rs:105:26
kablam!     |
kablam! 105 |                 Ok(_) => return fail!((ErrorKind::PatternError, "Could not read enough bytes")),
kablam!     |                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
kablam! 
kablam! warning: unreachable expression, #[warn(unreachable_code)] on by default
kablam!    --> src/lib.rs:7:23
kablam!     |
kablam! 7   |           if !($expr) { return $err_result; }
kablam!     |                         ^
kablam!     | 
kablam!    ::: src/script.rs
kablam!     |
kablam! 173 |                           ensure!(self.script.hash == hash,
kablam!     |  _________________________- starting here...
kablam! 174 | |                                 fail!((ErrorKind::BusyLoadingError, "load hash error")));
kablam!     | |_________________________________________________________________________________________- ...ending here: in this macro invocation
kablam! 
kablam! warning: unreachable expression, #[warn(unreachable_code)] on by default
kablam!    --> src/lib.rs:7:23
kablam!     |
kablam! 7   |           if !($expr) { return $err_result; }
kablam!     |                         ^
kablam!     | 
kablam!    ::: src/script.rs
kablam!     |
kablam! 184 |                           ensure!(self.script.hash == hash,
kablam!     |  _________________________- starting here...
kablam! 185 | |                                 fail!((ErrorKind::BusyLoadingError, "load hash error")));
kablam!     | |_________________________________________________________________________________________- ...ending here: in this macro invocation
kablam! 
kablam!     Finished debug [unoptimized + debuginfo] target(s) in 8.74 secs
boom! running `docker rm -f 91b4da1b3c0e901dec4b3b85ec79331eb6113ed40d1d7313d68603e3e5db3454`
blam! 91b4da1b3c0e901dec4b3b85ec79331eb6113ed40d1d7313d68603e3e5db3454
boom! running: cargo +stable test --frozen
boom! creating container for: cargo +stable test --frozen
boom! running `docker create -v /home/ec2-user/cargobomb/./work/local/test-source/nightly/stable:/source:ro -v /home/ec2-user/cargobomb/./work/local/cargo-home:/cargo-home:ro -v /home/ec2-user/cargobomb/./work/local/rustup-home:/rustup-home:ro -v /home/ec2-user/cargobomb/./work/local/target-dirs/nightly/stable:/target:rw -e USER_ID=500 -e CMD=cargo +stable test --frozen cargobomb`
blam! 9215abf5bb896019cd9323a5697a2425aada9c1d79589231ea59afbbaa88ff4b
boom! running `docker start -a 9215abf5bb896019cd9323a5697a2425aada9c1d79589231ea59afbbaa88ff4b`
kablam!     Finished debug [unoptimized + debuginfo] target(s) in 0.0 secs
kablam!      Running /target/debug/deps/td_rredis-9b82755f818d93ef
blam! 
blam! running 0 tests
blam! 
blam! test result: ok. 0 passed; 0 failed; 0 ignored; 0 measured
blam! 
kablam!      Running /target/debug/deps/test_basic-9be96ea8c983168b
blam! 
blam! running 25 tests
blam! test test_cluster ... FAILED
blam! test test_auto_m_versions ... FAILED
blam! test test_args ... FAILED
blam! test test_empty_pipeline ... FAILED
blam! test test_filtered_scanning ... FAILED
blam! test test_getset ... FAILED
blam! test test_info ... FAILED
blam! test test_hash_ops ... FAILED
blam! test test_incr ... FAILED
blam! test test_nice_api ... FAILED
blam! test test_nice_hash_api ... FAILED
blam! test test_nice_list_api ... FAILED
blam! test test_optionals ... FAILED
blam! test test_pipeline ... FAILED
blam! test test_pipeline_transaction ... FAILED
blam! test test_pubsub ... FAILED
blam! test test_real_transaction ... FAILED
blam! test test_real_transaction_highlevel ... FAILED
blam! test test_scan ... FAILED
blam! test test_scanning ... FAILED
blam! test test_tuple_args ... FAILED
blam! test test_script ... FAILED
blam! test test_set_ops ... FAILED
blam! test test_tuple_decoding_regression ... FAILED
blam! test test_invalid_protocol ... ok
blam! 
blam! failures:
blam! 
kablam! error: test failed
blam! ---- test_cluster stdout ----
blam! 	thread 'test_cluster' panicked at 'called `Result::unwrap()` on an `Err` value: Connection refused (os error 111)', /buildslave/rust-buildbot/slave/stable-dist-rustc-linux/build/src/libcore/result.rs:837
blam! note: Run with `RUST_BACKTRACE=1` for a backtrace.
blam! 
blam! ---- test_auto_m_versions stdout ----
blam! 	thread 'test_auto_m_versions' panicked at 'called `Result::unwrap()` on an `Err` value: Connection refused (os error 111)', /buildslave/rust-buildbot/slave/stable-dist-rustc-linux/build/src/libcore/result.rs:837
blam! 
blam! ---- test_args stdout ----
blam! 	thread 'test_args' panicked at 'called `Result::unwrap()` on an `Err` value: Connection refused (os error 111)', /buildslave/rust-buildbot/slave/stable-dist-rustc-linux/build/src/libcore/result.rs:837
blam! 
blam! ---- test_empty_pipeline stdout ----
blam! 	thread 'test_empty_pipeline' panicked at 'called `Result::unwrap()` on an `Err` value: Connection refused (os error 111)', /buildslave/rust-buildbot/slave/stable-dist-rustc-linux/build/src/libcore/result.rs:837
blam! 
blam! ---- test_filtered_scanning stdout ----
blam! 	thread 'test_filtered_scanning' panicked at 'called `Result::unwrap()` on an `Err` value: Connection refused (os error 111)', /buildslave/rust-buildbot/slave/stable-dist-rustc-linux/build/src/libcore/result.rs:837
blam! 
blam! ---- test_getset stdout ----
blam! 	thread 'test_getset' panicked at 'called `Result::unwrap()` on an `Err` value: Connection refused (os error 111)', /buildslave/rust-buildbot/slave/stable-dist-rustc-linux/build/src/libcore/result.rs:837
blam! 
blam! ---- test_info stdout ----
blam! 	thread 'test_info' panicked at 'called `Result::unwrap()` on an `Err` value: Connection refused (os error 111)', /buildslave/rust-buildbot/slave/stable-dist-rustc-linux/build/src/libcore/result.rs:837
blam! 
blam! ---- test_hash_ops stdout ----
blam! 	thread 'test_hash_ops' panicked at 'called `Result::unwrap()` on an `Err` value: Connection refused (os error 111)', /buildslave/rust-buildbot/slave/stable-dist-rustc-linux/build/src/libcore/result.rs:837
blam! 
blam! ---- test_incr stdout ----
blam! 	thread 'test_incr' panicked at 'called `Result::unwrap()` on an `Err` value: Connection refused (os error 111)', /buildslave/rust-buildbot/slave/stable-dist-rustc-linux/build/src/libcore/result.rs:837
blam! 
blam! ---- test_nice_api stdout ----
blam! 	thread 'test_nice_api' panicked at 'called `Result::unwrap()` on an `Err` value: Connection refused (os error 111)', /buildslave/rust-buildbot/slave/stable-dist-rustc-linux/build/src/libcore/result.rs:837
blam! 
blam! ---- test_nice_hash_api stdout ----
blam! 	thread 'test_nice_hash_api' panicked at 'called `Result::unwrap()` on an `Err` value: Connection refused (os error 111)', /buildslave/rust-buildbot/slave/stable-dist-rustc-linux/build/src/libcore/result.rs:837
blam! 
blam! ---- test_nice_list_api stdout ----
blam! 	thread 'test_nice_list_api' panicked at 'called `Result::unwrap()` on an `Err` value: Connection refused (os error 111)', /buildslave/rust-buildbot/slave/stable-dist-rustc-linux/build/src/libcore/result.rs:837
blam! 
blam! ---- test_optionals stdout ----
blam! 	thread 'test_optionals' panicked at 'called `Result::unwrap()` on an `Err` value: Connection refused (os error 111)', /buildslave/rust-buildbot/slave/stable-dist-rustc-linux/build/src/libcore/result.rs:837
blam! 
blam! ---- test_pipeline stdout ----
blam! 	thread 'test_pipeline' panicked at 'called `Result::unwrap()` on an `Err` value: Connection refused (os error 111)', /buildslave/rust-buildbot/slave/stable-dist-rustc-linux/build/src/libcore/result.rs:837
blam! 
blam! ---- test_pipeline_transaction stdout ----
blam! 	thread 'test_pipeline_transaction' panicked at 'called `Result::unwrap()` on an `Err` value: Connection refused (os error 111)', /buildslave/rust-buildbot/slave/stable-dist-rustc-linux/build/src/libcore/result.rs:837
blam! 
blam! ---- test_pubsub stdout ----
blam! 	thread 'test_pubsub' panicked at 'called `Result::unwrap()` on an `Err` value: Connection refused (os error 111)', /buildslave/rust-buildbot/slave/stable-dist-rustc-linux/build/src/libcore/result.rs:837
blam! 
blam! ---- test_real_transaction stdout ----
blam! 	thread 'test_real_transaction' panicked at 'called `Result::unwrap()` on an `Err` value: Connection refused (os error 111)', /buildslave/rust-buildbot/slave/stable-dist-rustc-linux/build/src/libcore/result.rs:837
blam! 
blam! ---- test_real_transaction_highlevel stdout ----
blam! 	thread 'test_real_transaction_highlevel' panicked at 'called `Result::unwrap()` on an `Err` value: Connection refused (os error 111)', /buildslave/rust-buildbot/slave/stable-dist-rustc-linux/build/src/libcore/result.rs:837
blam! 
blam! ---- test_scan stdout ----
blam! 	thread 'test_scan' panicked at 'called `Result::unwrap()` on an `Err` value: Connection refused (os error 111)', /buildslave/rust-buildbot/slave/stable-dist-rustc-linux/build/src/libcore/result.rs:837
blam! 
blam! ---- test_scanning stdout ----
blam! 	thread 'test_scanning' panicked at 'called `Result::unwrap()` on an `Err` value: Connection refused (os error 111)', /buildslave/rust-buildbot/slave/stable-dist-rustc-linux/build/src/libcore/result.rs:837
blam! 
blam! ---- test_tuple_args stdout ----
blam! 	thread 'test_tuple_args' panicked at 'called `Result::unwrap()` on an `Err` value: Connection refused (os error 111)', /buildslave/rust-buildbot/slave/stable-dist-rustc-linux/build/src/libcore/result.rs:837
blam! 
blam! ---- test_script stdout ----
blam! 	thread 'test_script' panicked at 'called `Result::unwrap()` on an `Err` value: Connection refused (os error 111)', /buildslave/rust-buildbot/slave/stable-dist-rustc-linux/build/src/libcore/result.rs:837
blam! 
blam! ---- test_set_ops stdout ----
blam! 	thread 'test_set_ops' panicked at 'called `Result::unwrap()` on an `Err` value: Connection refused (os error 111)', /buildslave/rust-buildbot/slave/stable-dist-rustc-linux/build/src/libcore/result.rs:837
blam! 
blam! ---- test_tuple_decoding_regression stdout ----
blam! 	thread 'test_tuple_decoding_regression' panicked at 'called `Result::unwrap()` on an `Err` value: Connection refused (os error 111)', /buildslave/rust-buildbot/slave/stable-dist-rustc-linux/build/src/libcore/result.rs:837
blam! 
blam! 
blam! failures:
blam!     test_args
blam!     test_auto_m_versions
blam!     test_cluster
blam!     test_empty_pipeline
blam!     test_filtered_scanning
blam!     test_getset
blam!     test_hash_ops
blam!     test_incr
blam!     test_info
blam!     test_nice_api
blam!     test_nice_hash_api
blam!     test_nice_list_api
blam!     test_optionals
blam!     test_pipeline
blam!     test_pipeline_transaction
blam!     test_pubsub
blam!     test_real_transaction
blam!     test_real_transaction_highlevel
blam!     test_scan
blam!     test_scanning
blam!     test_script
blam!     test_set_ops
blam!     test_tuple_args
blam!     test_tuple_decoding_regression
blam! 
blam! test result: FAILED. 1 passed; 24 failed; 0 ignored; 0 measured
blam! 
boom! running `docker rm -f 9215abf5bb896019cd9323a5697a2425aada9c1d79589231ea59afbbaa88ff4b`
blam! 9215abf5bb896019cd9323a5697a2425aada9c1d79589231ea59afbbaa88ff4b
